client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -131,7 +139,19 @@ class PlanningListSubNavComponent extends React.Component<IProps, IState> {
         planningApi.ui.list.search({sort_field: field});
     }
 
+    filterCoverageUser(item) {
+        planningApi.ui.list.search({coverage_user_id: item._id, coverage_user_name: item.display_name});
+    }
+
     render() {
+        let newOption = {_id: null, display_name: 'ALL'};
+        let List = [newOption, ...this.props.users];
```

Comment:
The naming convention we use in Typescript is CamelCase with the first character to be lower case